src/Simulation/qdk_sim_rs/rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly-2022-08-01
```

Comment:
How often will we need to update this?

Reply:
Strictly speaking, we don't need to on any fixed schedule — old nightly builds persist and are valid, but will not include bugfixes from future nightlies. From that perspective, it's good practice to update as part of the regular maintenance of the project. Since we use a minimal number of unstable features (specifically, only backtrace), the cost to updating is very minimal — everything not gated on a feature attribute is still stable and subject to API stability rules.